grafana-plugin/src/plugin.json

```diff
@@ -174,9 +174,10 @@
           { "action": "grafana-oncall-app.user-settings:admin" },
 
           { "action": "grafana-oncall-app.other-settings:read" },
-          { "action": "grafana-oncall-app.other-settings:write" }
-        ],
-        "hidden": true
```

Comment:
`hidden` is no longer supported. See https://raw.githubusercontent.com/grafana/grafana/main/docs/sources/developers/plugins/plugin.schema.json

(the Admin role is already _always_ shown in the RBAC role picker in the Grafana UI)